ArmNavigation/two_joint_arm_to_point_control/two_joint_arm_to_point_control.py

```diff
@@ -54,11 +59,17 @@ def two_joint_arm(GOAL_TH=0.0, theta1=0.0, theta2=0.0):
             theta2 = theta2 + Kp * ang_diff(theta2_goal, theta2) * dt
         except ValueError as e:
             print("Unreachable goal")
+        except TypeError:
+            x = x_prev
+            y = y_prev
 
         wrist = plot_arm(theta1, theta2, x, y)
 
         # check goal
-        d2goal = np.hypot(wrist[0] - x, wrist[1] - y)
+        try:
+            d2goal = np.hypot(wrist[0] - x, wrist[1] - y)
+        except TypeError:
+            pass
```

Comment:
This empty except is needed?

Reply:
Hi, thank you for your review. Yes, if this error is not passed, the same bug appears again as the value of x and y are "NoneType". I feel it is needed,

Reply:
OK. So could you please use if statement instead of try catch.